CorsixTH/Lua/epidemic.lua

```diff
@@ -695,8 +705,10 @@ function Epidemic:hasNoInfectedPatients()
 end
 
 function Epidemic:tryAnnounceInspector()
+  if not self.coverup_in_progress or not self:inspectorSpawned() then return end
+
   local inspector = self.inspector
-  if inspector and not inspector.has_been_announced and
+  if inspector.has_been_announced and
```

Comment:
Line 711 and 714 don't make sense together. I think you mean `if not inspector.has_been_announced`?

Reply:
> Line 711 and 714 don't make sense together. I think you mean `if not inspector.has_been_announced`?

Nice catch. thanks! Fixed